src/lib/core/CHIPConfig.h

```diff
@@ -1457,6 +1457,17 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #endif // CHIP_CONFIG_TEST
 #endif // CHIP_CONFIG_MAX_SCENES_TABLE_SIZE
 
+/**
+ * @def CHIP_CONFIG_MAX_SCENES
+ *
+ * @brief This defines enables the caude for the default scene handlers in each sceneable cluster.
```

Comment:
s/caude/clause/? 

or something else? I am not sure I can parse this sentence.

Reply:
Reworded